.github/workflows/ci.yml

```diff
@@ -197,11 +220,21 @@ jobs:
           tar -C tools/clang/dsymutil/ -xvzf dsymutil-llvmorg-17-init-19076-g5533fc10-1.tgz
           V8_FROM_SOURCE=true RUSTFLAGS="-C opt-level=1 -Zsanitizer=address" cargo +nightly -Z build-std test --lib --bins --tests -vv --target ${{ matrix.config.target }}
 
-      - name: Test
+      - name: Test -- No Pointer Compression
+        env:
+            SCCACHE_IDLE_TIMEOUT: 0
+        if: matrix.config.variant == 'debug' || matrix.config.variant == 'release'|| matrix.config.pointer_compression == 'false'
+        run:
+          ${{ matrix.config.cargo }} test -vv --all-targets --locked ${{ env.CARGO_VARIANT_FLAG }}
+          --target ${{ matrix.config.target }}
+
+      - name: Test -- With Pointer Compression
         env:
           SCCACHE_IDLE_TIMEOUT: 0
-        if: matrix.config.variant == 'debug' || matrix.config.variant == 'release'
+        if: matrix.config.variant == 'debug' || matrix.config.variant == 'release' || matrix.config.pointer_compression == 'true'
```

Comment:
I think you should make it a feature flag, and update build.rs to pass that flag to `gn`. 
See https://github.com/denoland/rusty_v8/blob/main/build.rs#L196-L198